README.md

```diff
@@ -21,5 +21,22 @@ To deploy an updated data version to the Agora development database
 To deploy an updated data version to the staging and prod database.
 1. Merge data-version update to staging and prod branches.
 
+# Setup
+
+The following environment variables need to be setup for the scripts to deploy database updates:
+
+| Variable             | Description                       | Example                                                                   |
+|----------------------|-----------------------------------|---------------------------------------------------------------------------|
+| BASTIAN_HOST_develop | The bastian host                  | ec2-10-11-12-13.compute-1.amazonaws.com                                   |
+| DB_HOST_develop      | The database host                 | dbcluster-mr0a782pfjnk.cluster-ctcayu3de2lt.us-east-1.docdb.amazonaws.com |
+| DB_USER_develop      | The database user                 | dbuser                                                                    |
+| DB_PASS_develop      | The database password             | supersecret                                                               |
+| SYNAPSE_USERNAME     | The Synapse service user          | syn-service-user                                                          |
+| SYNAPSE_PASSWORD     | The Synapse service user password | supersecret                                                               |
+
+__Note__: The variables containing `_develop` postfix corresponds to the branch.
+To deploy to a prod environment a prod branch is require along with a variable
```

Comment:
Should we mention the staging env and branch here too?

Reply:
nah, it's just an example. i think you get the jist 😄